qa/rpc-tests/bip65-cltv-p2p.py

```diff
@@ -42,6 +42,9 @@ def __init__(self):
         self.num_nodes = 1
 
     def setup_network(self):
+        #This test requires mocktime
```

Comment:
I don't understand why this test requires mocktime -- I just tested locally and it runs fine if I remove the changes you made to this file.  And logically I don't understand how this test can fail based on the code change in this PR?  I don't believe any transactions are sent over the network in this test (only put into blocks).